project/Dependencies.scala

```diff
@@ -228,6 +228,7 @@ object Dependencies {
   val Jms = Seq(
     libraryDependencies ++= Seq(
       "javax.jms" % "jms" % "1.1" % Provided, // CDDL + GPLv2
+      "com.ibm.mq" % "com.ibm.mq.allclient" % "9.1.1.0" % Test, // IBM International Program License Agreement https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/messaging/mqdev/maven/licenses/L-APIG-AZYF2E/LI_en.html
```

Comment:
We'll need to check the implications of that license. It might take a few days to get an answer.

Reply:
We don't see a problem with the "IBM International Program License Agreement".

Reply:
Do I need dod rebase?

Reply:
@ennru do I need make `merge` (or rebase) ?

Reply:
No, we'll review, and squash and merge if everything looks good.